test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1691,9 +1696,24 @@ def test_multiple_job_preemption_order(self):
         # Delete one running, one suspended job and one of high priority job
         # This will leave 2 free nodes to pick up the suspended and queued job
 
-        self.server.deljob(jid1)
-        self.server.deljob(jid2)
-        self.server.deljob(jid5)
+        self.server.deljob([jid1, jid2, jid5])
+
+        workq_dict = self.server.status(QUEUE, id='workq')[0]
+        expressq_dict = self.server.status(QUEUE, id='expressq')[0]
+
+        x = 1
+        while x < 20:
+            if workq_dict.get('total_jobs') == '2' and \
+                            expressq_dict.get('total_jobs') == '1':
```

Comment:
The status() needs to be inside this loop or these dictionaries will never change.